src/api/index.js

```diff
@@ -0,0 +1,15 @@
+async function getMovieList({ page }) {
+  try {
+    // https://api.themoviedb.org/3/movie/popular?api_key=252a4eaee3a54b8671ca4de1dd773f8d&page=1&language=ko-KR&sort_by=popularity.desc
+    const response = await fetch(
+      `${process.env.TMDB_BASE_URL}?page=${page}&api_key=${process.env.TMDB_API_KEY}&language=ko-KR&sort_by=popularity.desc`
+    );
+    const jsonData = await response.json();
+
+    return jsonData;
+  } catch (error) {
+    throw new Error(error);
+  }
```

Comment:
```suggestion
    // https://api.themoviedb.org/3/movie/popular?api_key=252a4eaee3a54b8671ca4de1dd773f8d&page=1&language=ko-KR&sort_by=popularity.desc
    const response = await fetch(
      `${process.env.TMDB_BASE_URL}?page=${page}&api_key=${process.env.TMDB_API_KEY}&language=ko-KR&sort_by=popularity.desc`
    );
    const jsonData = await response.json();

    return jsonData;
```
try/catch 자체가 불필요해보이네요!
어차피 에러를 정제하는게 아니라 그대로 다시 발생시키는거라서요 ㅎㅎ

Reply:
에러 관련된 코드를 보완해 봤습니다. 피드백 주시면 감사하겠습니다.